### PR TITLE
Language translation support

### PR DESCRIPTION
**Pull request by proudust**
Add translator support.

- Add translatable strings mark `_("~")`
- Add calendar translate support

### Add translatable strings mark

When you add a mark, it is added to `dialogue.tab` generated by Ren'Py Extract Dialogue function.
I want to add a mark to the `Event(prompt="~")`, but the PR was too big, so we refrained from it.

### Add calendar translate support

Make calendar months, days and events translatable.
`DATE_DISPLAY_FORMAT` needs to be fine-tuned depending on the font.

### CODE REVIEW

**Comment by multimokia** [comment]
Please rebase to the `content` branch. We don't use `enhancement` anymore

**Comment by proudust** [comment]
@multimokia sorry, fixed it

File: Monika After Story/game/screens.rpy [hunk 1]
```diff
@@ -3,34 +3,30 @@ init 100 python:
...
-        " or considered tasteless."
+    layout.MAS_TT_SENS_MODE = _(
+        "Sensitive mode removes content that may be disturbing, offensive, or considered tasteless."
     )
-    layout.MAS_TT_UNSTABLE = (
-        "Unstable mode downloads updates from the experimental unstable "
-        "branch of development. It is HIGHLY recommended to make a backup "
-        "of your persistents before enabling this mode."
+    layout.MAS_TT_UNSTABLE = _(
+        "Unstable mode downloads updates from the experimental unstable branch of development. It is HIGHLY recommended to make a backup of your persistents before enabling this mode."
```
Comment (ThePotatoGuy) [inline Monika After Story/game/screens.rpy]: long string lines are split to keep the character line limit to 80-120 characters. please split these lines like they were before.

File: Monika After Story/game/screens.rpy [hunk 2]
```diff
@@ -3,34 +3,30 @@ init 100 python:
...
-    layout.QUIT_NO = "Thank you, [player]!\nLet's spend more time together~"
+    layout.QUIT_YES = _("Please don't close the game on me!")
+    layout.QUIT_NO = _("Thank you, [player]!\nLet's spend more time together~")
 
     # tooltips
-    layout.MAS_TT_SENS_MODE = (
-        "Sensitive mode removes content that may be disturbing, offensive, "
-        " or considered tasteless."
+    layout.MAS_TT_SENS_MODE = _(
+        "Sensitive mode removes content that may be disturbing, offensive, or considered tasteless."
```
Comment (ThePotatoGuy) [inline Monika After Story/game/screens.rpy]: long string lines are split to keep the character line limit to 80-120 characters. please split these lines like they were before.

File: Monika After Story/game/screens.rpy [hunk 3]
```diff
@@ -3,34 +3,30 @@ init 100 python:
...
     )
-    layout.MAS_TT_G_NOTIF = (
+    layout.MAS_TT_G_NOTIF = _(
         "Enables notifications for the selected group."
     )
-    layout.MAS_TT_ACTV_WND = (
-        "Enabling this will allow Monika to see your active window "
-        "and offer some comments based on what you're doing."
+    layout.MAS_TT_ACTV_WND = _(
+        "Enabling this will allow Monika to see your active window and offer some comments based on what you're doing."
```
Comment (ThePotatoGuy) [inline Monika After Story/game/screens.rpy]: long string lines are split to keep the character line limit to 80-120 characters. please split these lines like they were before.

File: Monika After Story/game/screens.rpy [hunk 4]
```diff
@@ -42,30 +38,27 @@ init python in mas_layout:
...
     QUIT_NO = store.layout.QUIT_NO
-    QUIT = "Leaving without saying goodbye, [player]?"
-    UNSTABLE = (
-        "WARNING: Enabling unstable mode will download updates from the " +
-        "experimental unstable branch. It is HIGHLY recommended to make a " +
-        "backup of your persistents before enabling this mode. Please report " +
-        "issues found here with an [[UNSTABLE] tag."
+    QUIT = _("Leaving without saying goodbye, [player]?")
+    UNSTABLE = _(
+        "WARNING: Enabling unstable mode will download updates from the experimental unstable branch. It is HIGHLY recommended to make a backup of your persistents before enabling this mode. Please report issues found here with an [[UNSTABLE] tag."
```
Comment (ThePotatoGuy) [inline Monika After Story/game/screens.rpy]: long string lines are split to keep the character line limit to 80-120 characters. please split these lines like they were before.

File: Monika After Story/game/zz_calendar.rpy [hunk 5]
```diff
@@ -3,6 +3,9 @@
 # Contains also a store named mas_calendar which includes helper functions
 # to add Events to the calendar
 
+# Format used for calendar display
+define DATE_DISPLAY_FORMAT = "\t  \t  \t  \t  \t  \t  \t  {0}\n{1}\n{2}\n{3}"
```
Comment (ThePotatoGuy) [inline Monika After Story/game/zz_calendar.rpy]: Why did you take this out of `MASCalendar`? This is extremely specific to that displayable and should remain in there.

File: Monika After Story/game/zz_calendar.rpy [hunk 6]
```diff
@@ -1760,9 +1760,9 @@ init python:
         ):
         calendar.addRepeatable_d(
             "player-bday",
-            "Your Birthday",
-            pbday,
-            range(pbday.year,MASCalendar.MAX_VIEWABLE_YEAR)
+            _("Your Birthday"),
+            persistent._mas_player_bday,
+            []
```
Comment (ThePotatoGuy) [inline Monika After Story/game/zz_calendar.rpy]: revert this back so its using `pbday` and a range.

**Comment by ThePotatoGuy** [comment]
@Legendkiller21 can you verify general stability (topics and stuff still work) and that calendar still looks ok?

**Pull request merged**